### ISSUE TITLE
Upgrade from 1.5.3 to 2.0.1 breaks deploy script

### ISSUE THREAD

**Issue by loed-idzinga** [rollbar/rollbar-gem#266]
Updated the Rollbar gem from 1.5.3 to 2.0.1. With this update, our deploy script breaks with the error below. `rake rollbar:test` works without a problem. This is in combination with Capistrano 2
.15.6 and `rollbar/capistrano`. 

```bash
/opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rollbar-2.0.1/lib/rollbar/json.rb:29:in `dump': undefined method `call' for nil:NilClass (NoMethodError)
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rollbar-2.0.1/lib/rollbar/capistrano.rb:33:in `block (3 levels) in load_into'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:138:in `instance_eval'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:138:in `invoke_task_directly'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/callbacks.rb:25:in `invoke_task_directly_with_callbacks'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:89:in `execute_task'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:101:in `find_and_execute_task'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/callback.rb:38:in `call'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/callbacks.rb:141:in `block in trigger'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/callbacks.rb:141:in `each'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/callbacks.rb:141:in `trigger'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/callbacks.rb:27:in `invoke_task_directly_with_callbacks'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:89:in `execute_task'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/configuration/execution.rb:101:in `find_and_execute_task'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/execute.rb:46:in `block in execute_requested_actions'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/execute.rb:45:in `each'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/execute.rb:45:in `execute_requested_actions'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/help.rb:19:in `execute_requested_actions_with_help'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/execute.rb:34:in `execute!'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/lib/capistrano/cli/execute.rb:14:in `execute'
	from /opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/capistrano-2.15.6/bin/cap:4:in `<top (required)>'
	from /opt/boxen/rbenv/versions/2.2.2/bin/cap:23:in `load'
	from /opt/boxen/rbenv/versions/2.2.2/bin/cap:23:in `<main>'
```

**Comment by jondeandres** [comment]
@loed-idzinga we'll release a new version in few minutes.

For now, if you want a fix, in `/opt/boxen/rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rollbar-2.0.1/lib/rollbar/capistrano.rb:33` change `JSON.dump(params)` with `::JSON.dump(params`.

We've added a `Rollbar::JSON` module, so we need to add the `::` there.

Sorry for the inconveniences and thanks for the report!

**Comment by loed-idzinga** [comment]
@jondeandres Nice, thanks for the quick response. I'll try the new release as soon as possible. Keep up the good work! :+1:
